ui/web/templates/registration.html

```diff
@@ -0,0 +1,157 @@
+<!-- Copyright (c) Abstract Machines
+SPDX-License-Identifier: Apache-2.0 -->
+
+{{ define "registration" }}
+  <!doctype html>
+  <html lang="en">
+    <head>
+      <title>Register</title>
+      {{ template "header" }}
+    </head>
+    <body class="login-body">
+      <div class="container mt-5 pt-5">
+        <div class="row p-5">
+          <div class="login-card col-lg-4 p-md-5 mx-auto mt-5">
```

Comment:
```suggestion
          <div class="login-card col-lg-4 p-md-5 mx-auto mt-4">
```

to remove the overflow occuring

Reply:
This is the top margin of the login card and will not change content from overflowing.

Content overflows because the text is bigger than the width of the container and it is perfectly normal. For example 
![image](https://github.com/absmach/magistrala-ui/assets/28790446/4d1ba86b-c0c0-4403-aece-f9bd06b7f93e)